### ISSUE TITLE
Cropped images not downloading

### ISSUE THREAD

**Issue by ShawnieBoy** [magarena/magarena#1154]
Many sets have been updated on Magiccards.info - however the cropped images are not downloading, or being listed as missing, in the setup images menu when Rendered Proxy is selected. The sets are:

Commander 2016, Consipracy: Take the Crown, Kaladesh and Aether Revolt.

**Comment by ShawnieBoy** [comment]
It looks as though if there is a full image already downloaded, it will not download a crop.

**Comment by ShawnieBoy** [comment]
Another thing I've noticed - your display preference on launch overrides the download count for your current setting. e.g. You have 'printed' as your setting, when you launch you have, say 100, images to download - if you change your display setting the count doesn't change (even if you have no missing images for crops in the example).

**Comment by lodici** [comment]
Yeah, I am still not 100% happy with the image downloading - it's complicated but I think it could be simpler from a UI POV.
